apps/angular/src/app/features/anime-table/anime-table.component.html

```diff
@@ -0,0 +1,45 @@
+<mat-toolbar color="primary">
+  <mat-toolbar-row>
+    <span id="page-heading">Anime Catalog</span>
+  </mat-toolbar-row>
+</mat-toolbar>
+<table
+  mat-table
+  *ngIf="anime$"
+  [dataSource]="anime$"
```

Comment:
Try this, i guess it should work. Also you can avoid subscribing inside your component this way.
```suggestion
  *ngIf="animeList$| async as animeList"
  [dataSource]="animeList"
```